scripts/xsdflatten.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+import sys
+import re
+import copy
+from lxml import etree
+
+
+def get_includes_from_file(filename):
+    pattern = re.compile('(<xs:include schemaLocation)')
```

Comment:
`<xs:include/>` can have [other attributes](https://www.w3schools.com/xml/el_include.asp). If one of those slides in before the `schemaLocation`, it could be missed. 

```xml
<xs:include id="something" schemaLocation="BaseElements.xsd">
```

I'd suggest using an xpath to find these instead:

```xpath
//xs:include/@schemaLocation
```

Reply:
Good call. For what it's worth, this code came from [here](https://github.com/esunder/xsdflatten).